GameData/RP-1/Contracts/Earth Observation 1/EOSCoronaRep.cfg

```diff
@@ -144,8 +138,8 @@ CONTRACT_TYPE
 				situation = InSpaceLow
 				experiment = RP0photos2
 				fractionComplete = 0.9
-				minSubjectsToComplete = 3 
-				title = Collect 90% of the available imagery across @requiredBiomes biomes
+				minSubjectsToComplete = 3 + @index
+				title = Collect 90% of the available imagery across @minSubjectsToComplete biomes

```

Comment:
Realized this math doesn't work.  Suggestions for starting percentage and number of biomes?